src/pages/transactions/[txid].page.tsx

```diff
@@ -152,6 +182,35 @@ function getDfTx(vouts: TransactionVout[]): DfTx<any> | undefined {
   return (stack[1] as OP_DEFI_TX).tx;
 }
 
+function getNetworkParams(network: EnvironmentNetwork): string {
+  switch (network) {
+    case EnvironmentNetwork.MainNet:
+      // no-op: network param not required for MainNet
+      return "";
+    case EnvironmentNetwork.TestNet:
+      return `?network=${EnvironmentNetwork.TestNet}`;
+    case EnvironmentNetwork.DevNet:
+      return `?network=${EnvironmentNetwork.DevNet}`;
+
+    case EnvironmentNetwork.LocalPlayground:
+    case EnvironmentNetwork.RemotePlayground:
+      return `?network=${EnvironmentNetwork.RemotePlayground}`;
+    case EnvironmentNetwork.Changi:
+      return `?network=${EnvironmentNetwork.Changi}`;
+    default:
+      return "";
+  }
+}
+
+function getMetaScanTxUrl(
+  network: EnvironmentNetwork,
+  id?: string | null,
+): string {
+  const baseMetaScanUrl = "https://meta.defiscan.live";
```

Comment:
same for getMetaScanTxUrl and getMetaScanBlockUrl, if in case we have to update the url then we have to update them in two places, instead lest just use config/env var for this ?